src/lib/lib3index/cholesky.cc

```diff
@@ -1,12 +1,7 @@
 /*
- * @BEGIN LICENSE
+ *@BEGIN LICENSE
```

Comment:
Ok, now I show this PR as up-to-date. I think you just needed the force push before.

I'm going to pull in the SAD PR since that's already up-to-date wrt upstream master and the Travis is good for it. So after that, will you rebase and force push this cholesky PR, triggering Travis. Then I'll pull this one in.